### ISSUE TITLE
VFS directories for tags not showing =value entries  in version 0.7.1

### ISSUE THREAD

**Issue by taiyu-len** [oniony/TMSU#148]
For example
```sh
$ touch foo
$ mkdir mnt
$ tmsu tag foo var=value
$ tmsu mount mnt
$ ls mnt/tags/var
# in 0.7.0 the output is
files =value var
# in 0.7.1 the output is
files

$ ls mnt/tags/var/files
# in 0.7.0 the output is
foo
# in 0.7.1 the output is nothing
```

**Comment by oniony** [comment]
Thanks for the bug report. I'll take a look.

**Comment by oniony** [comment]
Confirmed broken in 0.7.1.

**Comment by oniony** [comment]
Seems this bug has been revealed by the bug fix of #141. I'll need to rework how this works.

**Comment by oniony** [comment]
For a bit more detail, the problem is that when you go to the path:

    mp/tags/sometag

This will retrieve all of the further tags for the set of files with the tag 'sometag'. Then TMSU takes this set of files and finds all of the values that can be applied to this set of files to produce further filesets. This is wrong.

The problem is that the set of files with 'colour=green' is not necessarily a subset of the files that have the tag 'colour'. So this logic is flawed. Instead TMSU shouldn't be determining the set of files on the file tag when evaluating values. It should be working at a level higher (which means the entire set of files in OP's case). Consider the following:

    mp/photos/year/=2018/colour

To work out which fileset to use when determining which colours to return, TMSU should be evaluating the set of files for 'photos and year=2018' and not 'photos and year=2018 and colour'. From that fileset, it should then determine which of the colours are appropriate.

**Comment by oniony** [comment]
Released v0.7.2 to encompass this fix.
